main.js

```diff
@@ -0,0 +1,430 @@
+$(document).ready(initializeGame);
+
+function initializeGame(){
+    attachClickHandlers();
+    currentPlayer = player[0];
+    $(".player1").find(".token1").hide();
+};
+function attachClickHandlers(){
+    $(".forfeitButton").on("click", resetGame);
+    $(".column").on("click", piecePlacement);
+
+};
+
+
+// GLOBAL VARIABLES
+
+var column_clicked = null;
+var row_clicked = null;
+var rowToPlacePiece = null;
+var putPieceInRow = 0;
+var games_played = 0;
+var attempts = 0;
+var piecesInColumn = 0;
+// ----------------
+
+
+var player = [];
+player[0] = {
+    name: "player1",
+    points: 0,
+    gamesWon: 0,
+    gamesLost: 0,
+    gamesTied: 0,
+};
+player[1] = {
+    name: "player2",
+    points: 0,
+    gamesWon: 0,
+    gamesLost: 0,
+    gamesTied: 0,
+};
+
+
+function placePlayerToken(piecesInRow) {
+    if (currentPlayer === player[0]) {
+        gameArray[piecesInRow][column_clicked].token = 1;
+    } else if (currentPlayer === player[1]) {
+        gameArray[piecesInRow][column_clicked].token = 2;
+    }
+}
+
+
+
+
+function displayToken(piecesInRow, column_clicked) {
+    if (gameArray[piecesInRow][column_clicked].token === 1) {
+        $(".row" + piecesInRow + " .column" + column_clicked).addClass("showPlayer1")
+
+    } else if (gameArray[piecesInRow][column_clicked].token === 2) {
+        $(".row" + piecesInRow + " .column" + column_clicked).addClass("showPlayer2")
+    }
+}
+
+    function changeTurn() {
+        if (currentPlayer === player[0]) {
+            currentPlayer = player[1];
+        } else if (currentPlayer === player[1]) {
+            currentPlayer = player[0];
+        }
+        ;
+    };
+
+    function piecePlacement() {
+        var jQueryObj = $(this);
+        columnIndex(jQueryObj);
+        var rowPieceShouldBePlaced = rowIndex();
+        placePlayerToken(rowToPlacePiece);
+        displayToken(rowPieceShouldBePlaced, column_clicked);   
+        var playerNumber = returnPlayerNumber(currentPlayer);
+        winCondition(rowToPlacePiece, column_clicked, gameArray, playerNumber);
+        changeTurn();
+        console.log('Piece placement function working');
+        console.log('This is: ', this);
+    }
+
+    function columnIndex(jQueryObj) {
+
+        column_clicked = jQueryObj.attr("id");
```

Comment:
setting a global variable like this can cause problems if something else also changes it for some reason.  It's like leaving something valuable on your lawn, hoping no one messes with it while you are gone.